### PR TITLE
update aio-script.sh

### PR DESCRIPTION
**Pull request by codetronaut**
Added Monitoring function(Task 3).

### CODE REVIEW

File: aio-script.sh [hunk 1]
```diff
@@ -32,9 +32,33 @@ elif [ -f /etc/redhat-release ];
     
  fi
 }
+
 function monitoring(){
 	echo "Monitoring..."
+if [ -f /etc/os-release ];
+    then
+        not_installed=$(dpkg -s htop 2>&1 | grep "install or installed")
```
Comment (amanharitsh123) [inline aio-script.sh]: Replace "or" with "ok".

File: aio-script.sh [hunk 2]
```diff
@@ -32,9 +32,33 @@ elif [ -f /etc/redhat-release ];
...
+        if [ -n "$not_installed" ];
```
Comment (amanharitsh123) [inline aio-script.sh]: Remove double quotes from $not_installed.

**Comment by codetronaut** [comment]
Updated the script please review it.

**Comment by amanharitsh123** [comment]
Great. Merging now.

**Pull request merged**